src/damage.cpp

```diff
@@ -190,7 +190,11 @@ void ammo_effects( const tripoint &p, const std::set<std::string> &effects )
             g->m.add_field( pt, fd_tear_gas, 3, 0 );
         }
     }
-
+    if( effects.count( "GAS_FUNGICIDAL" ) > 0 ) {
+        for( auto &&pt : g->m.points_in_radius( p, 1, 0 ) ) {
+            g->m.add_field( pt, fd_fungicidal_gas, 4, 0 );
```

Comment:
Looks like you're adding a field with density 4. This will be ignored: density is capped at 3 at the moment.